src/lib/Map.svelte

```diff
@@ -89,13 +111,55 @@
 			// Change the cursor to a pointer when the mouse is over the moments layer.
 			map.on('mouseenter', 'moments-layer', function () {
 				map.getCanvas().style.cursor = 'pointer';
+			
+				openPopup = false;
 			});
-
+			
 			// Change it back to a pointer when it leaves.
 			map.on('mouseleave', 'moments-layer', function () {
 				map.getCanvas().style.cursor = '';
+				openPopup = true;
 			});
 		});
+
+		const app = new SvelteToast({
```

Comment:
I think this is not needed. Removing it doesn't seem to change anything. Then also remove the import.